server/server.go

```diff
@@ -1583,6 +1584,18 @@ func PrintServerAndExit() {
 	os.Exit(0)
 }
 
+// printBuildinfoAndExit will print out https://pkg.go.dev/runtime/debug#BuildInfo and exit.
+func printBuildinfoAndExit() {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok || bi == nil {
+		//nolint:forbidigo
+		fmt.Println("failed to read Buildinfo")
```

Comment:
maybe instead something like this:
```suggestion
		fmt.Fprintln(os.Stderr, "failed to read BuildInfo")
```